_apps/vaquita.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Vaquita, fast and Accurate Identification of Structural Variation Using Combined Evidence
+layout: app
+contact: Jongkyu Kim
+categories: official
+links:
+  website: https://github.com/seqan/vaquita
+  download: https://github.com/seqan/vaquita
+  source: https://github.com/seqan/vaquita
+  doi: https://drops.dagstuhl.de/opus/volltexte/2017/7635/
```

Comment:
```suggestion
  doi: https://doi.org/10.4230/LIPIcs.WABI.2017.13
```

But I think it should rather be added as citation?

Reply:
Because citations already link and now every app has an empty DOI field

Reply:
I'll remove it again